pkgs/by-name/ni/nix-editor/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, darwin }:
+
+rustPlatform.buildRustPackage {
+  pname = "nix-editor";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "snowfallorg";
+    repo = "nix-editor";
+    rev = "b5017f8d61753ce6a3a1a2aa7e474d59146a8ae3";
+    hash = "sha256-Ne9NG7x45a8aJyAN+yYWbr/6mQHBVVkwZZ72EZHHRqw=";
+  };
+
+  cargoHash = "sha256-ZWSLW+mFT+Qls2GeVPamispL8ObieXT4tDc1Go5cjL4=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ darwin.Security ];
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
I am confused whats the correct convention now. I know `with` is an antipattern, but it seems it is okay for meta?
grepping for `meta = with lib;` yields 2.1k where `meta = {` yields 1.7k results.
Whats the way to go ?

Reply:
It's because human readers misunderstand bindings that escape `with` within `meta`, which can turn into errors when converting to `finalAttrs` (for example, `version` is present both under `lib` and the derivation attrset, as discussed in #292468).

I plan to do a treewide at some point to get rid of `meta = with lib;`

Reply:
I guess if the treewide is coming then this is less important. marking resolved